internal/server/list_documents.go

```diff
@@ -32,8 +34,10 @@ func renderTemplateForDocuments(client DocumentsClient, tmpl Template) Handler {
 		}
 
 		ctx := getContext(r)
+		urlParams := r.URL.Query()
+		sort := urlbuilder.CreateSortFromURL(urlParams, []string{"receiveddatetime"})
 
-		documentList, err := client.GetDeputyDocuments(ctx, app.DeputyId())
+		documentList, err := client.GetDeputyDocuments(ctx, app.DeputyId(), fmt.Sprintf("%s:%s", sort.OrderBy, "desc"))
```

Comment:
Does this `GetDeputyDocuments` get called anywhere else?
If not rather than passing it into the sirius level from here could you just not add it in directly ie
`req, err := c.newRequest(ctx, http.MethodGet, fmt.Sprintf("/api/v1/persons/%d/documents?&sort=receiveddatetime:desc", deputyId), nil)`

Reply:
I did it this way as I thought it would be cleaner when we add in the filters later on. Happy to change it if you don't think this will be used when we do the filtering.